### DISCUSSION TITLE
Welcome to opengeos.github.io Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/opengeos.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## How do I add a new repo to opengeos?

1. Create the repository elsewhere, such as under your personal account.
2. File a ticket [here](https://github.com/opengeos/opengeos.github.io/issues) asking to join the organization, including a link to the repository being added.
3. Accept the invitation to join the organization.
4. Use the GitHub UI to move the repository into the organization (you will need administrator/owner rights on the repository to do this). See [the GitHub documentation on repository transfer](https://help.github.com/en/github/administering-a-repository/transferring-a-repository) for details.
5. After it is moved, rename the repository as needed. Don't forget to update the documentation in the repository, too!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by christianbaez78** [comment]
Thanks for creating this amazing group. I am eager to learn more about programation and mapping!!

**Comment by slowy07** [comment]
its need requirement to join opengeos community sir @giswqs ? thanks!

**Reply by giswqs** [reply]
@slowy07 You mean you want to join the GitHub organization? I can add you if you are interested.

**Reply by slowy07** [reply]
yes of course sir @giswqs , i interseted about join opengeos by maaintain some repositories to make sure the code is clean and more readable and more efficient

**Comment by youssefmejridjo** [comment]
yes add me

**Comment by NerisVold** [comment]
Gracias por crear este espacio. Me interesan las herramientas geoespaciales de código abierto, la teledetección, los flujos de trabajo cartográficos y la automatización SIG con Python. Espero utilizar esta comunidad para aprender de otros colaboradores, plantear preguntas técnicas y compartir ejemplos útiles cuando sea posible. Mafia Casino parte de una temática fuerte, reconocible y con mucho potencial narrativo. Conviene que Mafia Casino utilice ese [https://mafiacasinologin.es/](https://mafiacasinologin.es/) ambiente con moderación, porque demasiados elementos oscuros podrían dificultar la navegación. Los pasos para transferir repositorios están claros y deberían facilitar que nuevos proyectos se unan a la organización sin confusiones. Me interesan especialmente las conversaciones sobre flujos de trabajo reproducibles, documentación, notebooks y herramientas que hagan el análisis geoespacial más accesible tanto para principiantes como para usuarios experimentados.
